.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v1
       with:
-        python-version: "3.10.4"
+        python-version: "3.10.10"
```

Comment:
Seems `setup-python` doesn't support version `3.10.4` anymore, so I bumped to a supported version.